src/main/java/io/jenkins/plugins/pipeline_log_fluentd_cloudwatch/CloudWatchAwsGlobalConfiguration.java

```diff
@@ -84,6 +104,53 @@ public void setLogGroupName(String logGroupName) {
         save();
     }
 
+    public String getFluentdHost() {
+        return fluentdHost;
+    }
+
+    @DataBoundSetter
+    public void setFluentdHost(String fluentdHost) {
+        this.fluentdHost = fluentdHost;
+    }
+
+    public int getFluentdPort() {
+        return fluentdPort;
+    }
+
+    /**
+     * @return the fluentd host calculated from configured values, environment variable
+     *         <code>FLUENTD_SERVICE_HOST</code> and default <code>localhost</code>
+     */
+    String computeFluentdHost() {
+        return computeFluentdHost(getFluentdHost());
+    }
+
+    /**
+     * @return the fluentd port calculated from configured values, environment variable
+     *         <code>FLUENTD_SERVICE_PORT_TCP</code> and default <code>24224</code> port
+     */
+    int computeFluentdPort() {
+        return computeFluentdPort(getFluentdPort());
+    }
+
+    private static String computeFluentdHost(String fluentdHost) {
+        String host = fluentdHost != null ? fluentdHost : System.getenv("FLUENTD_SERVICE_HOST");
```

Comment:
those environment variables are set by k8s when running fluentd in there, so it is useful for autoconfiguration

Reply:
In that case please at least use these as the default values for the getter, so that the config form never shows blank or zero.

Reply:
if the config form shows the env var values, and the form is saved then the field would no longer be empty and the env vars would not be used. It would set explicit values, or am I missing something?